lib/iris/tests/unit/coords/test_Coord.py

```diff
@@ -354,5 +355,13 @@ def test_different_array_attrs_incompatible(self):
         self.assertFalse(self.test_coord.is_compatible(self.other_coord))
 
 
+class Test_convert_units(tests.IrisTest):
+    def test_convert_unknown_units(self):
+        coord = iris.coords.AuxCoord(1, units='unknown')
+        emsg = ('Cannot convert from unknown units. '
+                'The "coord.units" attribute may be set directly.')
+        with self.assertRaisesRegexp(UnitConversionError, emsg):
+            coord.convert_units('degrees')
+
```

Comment:
Tests are probably going to freak out about the PEP8 of your newlines here...

Reply:
I have run the `pep8` command on my local copy of that file and got no output.